### ISSUE TITLE
3.1.2 `uninitialized constant Pango::ALIGN_LEFT`

### ISSUE THREAD

**Issue by andymeneely** [ruby-gnome2/ruby-gnome2#1022]
A bunch of constants appear to be missing upon the 3.1.2 release. This worked on 3.1.1. 

https://travis-ci.org/andymeneely/squib/jobs/222472071

You can see that the same code built just fine on the same commit but with 3.1.1: 

https://travis-ci.org/andymeneely/squib/builds/222471188

Is this part of the "GBytes to String conversion" in the changelog - I'm not sure what that means.

**Comment by kou** [comment]
Thanks for your report.
pango gem has some unexpected backward incompatibilities. We'll release a new version. With the new version, your test will be pass with `Pango::Layout` -> `Pango` substitutions against `spec/data/samples/*/*.txt` like the following:

```diff
diff --git a/spec/data/samples/autoscale_font/_autoscale_font.rb.txt b/spec/data/samples/autoscale_font/_autoscale_font.rb.txt
index a76c92a..2b1e904 100644
--- a/spec/data/samples/autoscale_font/_autoscale_font.rb.txt
+++ b/spec/data/samples/autoscale_font/_autoscale_font.rb.txt
@@ -64,9 +64,9 @@ pango font: size=([32768])
 pango: font_description=([MockDouble])
 pango: text=(["Short & Big"])
 pango: width=([286720])
-pango: wrap=([#<Pango::Layout::WrapMode word-char>])
-pango: ellipsize=([#<Pango::Layout::EllipsizeMode none>])
-pango: alignment=([#<Pango::Layout::Alignment center>])
+pango: wrap=([#<Pango::WrapMode word-char>])
+pango: ellipsize=([#<Pango::EllipsizeMode none>])
+pango: alignment=([#<Pango::Alignment center>])
 pango: justify=([false])
 cairo: move_to([0, 0])
 cairo: move_to([0, 0])
```

**Comment by kou** [comment]
I've released 3.1.3.
